src/regridStates/gauIncr2native.nml

```diff
@@ -0,0 +1,31 @@
+&config
+ n_vars=4,                             ! number of vars in list below
+ variable_list(1)="soilt1_inc     ", 
```

Comment:
I don't think we need the space in "soilt1_inc"

Also var list in one line might be better (personal preference to save space) 
variable_list="soilt1_inc","slc1_inc","",

Reply:
We do need it. Here's the gsi [output](https://github.com/NOAA-EMC/GSI/blob/092125196a49e856ad760730edac01e398a84d8f/src/enkf/gridio_gfs.f90#L4141)

Reply:
That makes more sense! I believe we do need the fixed length for specifying a char array in a namelist, but will double check.

Reply:
It worked! I changed it.